src/extensions/scratch3_music/index.js

```diff
@@ -112,14 +112,19 @@ class Scratch3MusicBlocks {
     _storeSound (filePath, index, bufferArray) {
         const fullPath = `${filePath}.mp3`;
 
-        if (!assetData[fullPath]) return;
+        return this._fetchSounds()
+            .catch(() => ({}))
```

Comment:
Yes. In case loading the sounds throws an error running the dynamic module or require.ensure is not available, this'll let the extension gracefully fail.